stardog-userrole-operator/test/clusterrole_test.go

```diff
@@ -0,0 +1,65 @@
+package test
```

Comment:
I would change the wording from `does not have correct name` to `has a wrong name`